packages/otoggl/otoggl.0.3.2/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Bindings for Toggl API in OCaml"
+description: "Bindings for Toggl API in OCaml"
+maintainer: ["Christophe Riolo Uusivaara"]
+authors: ["Christophe Riolo Uusivaara"]
+license: "MIT"
+homepage: "https://github.com/unitrack-time-tracking/OToggl"
+bug-reports: "https://github.com/unitrack-time-tracking/OToggl/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "atdgen" {build & >= "2"}
+  "atdgen-runtime" {>= "2"}
+  "base64" {>= "3"}
+  "containers" {>= "3.6"}
+  "dune" {>= "2.9" & >= "2.0"}
```

Comment:
```suggestion
  "dune" {>= "2.9"}
```
To fix this you just have to remove dune from the dependencies in the dune-project file (it is added automatically)